### ISSUE TITLE
No real time UPDATE Block Limit

### ISSUE THREAD

**Issue by Bricec6** [BentoBoxWorld/Limits#69]
**Description**
Hello, when a permission bskyblock.island.limit.hopper.40 is given, while the default limit is 20, the addons don't immediately calculate that a permission to increase the limit a was given, so it stays at 20 limit for the player BUT if the player disconnects and reconnects, the permission bskyblock.island.limit.hopper.40 will work and the player will be limited to 40.

Ex:
Limit > 20
Add perm to the creator of the island: bskyblock.island.limit.hopper.40
Limit > 20
If disconnected/reconnect: Limit > 40

Trad: www.DeepL.com

 - MARIADB
 - OS: WINDOW 10
 - BentoBox version: 1.11.0
 - Addons installed?
https://i.imgur.com/NnliDz7.png

Before it seems to me that this problem does not exist because I had made conclusive tests, now the bug came in the recent versions.

**Comment by tastybento** [comment]
This is true - we check permissions on login and then store them. Checking permissions is quite complex and running that code every single time a block is placed or an entity spawned was too much I thought so I cache the values. So, this is not a bug but a limitation: players need to relog to get the permission change benefit.
